docs/extensions/server/configuration.rst

```diff
@@ -18,7 +18,52 @@ Inject Session Conf with Custom Config Advisor
 
 .. versionadded:: 1.5.0
 
-.. caution:: unstable
+The step of inject session conf
+-------------------
 
-.. warning::
-   This page is still in-progress.
+1. create a custom class which implements the `org.apache.kyuubi.plugin.SessionConfAdvisor`.
+2. adding configuration at `kyuubi-defaults.conf`:
+
+   .. code-block:: java
+
+      kyuubi.session.conf.advisor=${classname}
+
+3. compile and put the jar into `$KYUUBI_HOME/jars`
+
+The `org.apache.kyuubi.plugin.SessionConfAdvisor` has a zero-arg constructor and holds one method with user and session conf and return a new conf map back.
```

Comment:
`returns`

Reply:
`and` holds -> `,` holds

Reply:
returns has the back meaning inside the word, we can remove the last redundant `back`.